File: culturio-src/web/users/models/userAuthDto.ts
export class UserAuthDto {
  public id: number;
  public externalId: string;
  public role: string;
  constructor(
    id: number,
    externalId: string,
    role: string
  ) {
    this.id = id;
    this.externalId = externalId;
    this.role = role;
  }
}

File: culturio-src/application/Users/Models/UserAuthDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Culturio.Application.Users.Models
{
    public class UserAuthDto
    {
        public int Id { get; set; }
        public string ExternalId { get; set; }
        public string Role { get; set; }
    }
}
